order/src/main/java/com/yas/order/repository/OrderRepository.java

```diff
@@ -16,7 +16,7 @@
 @Repository
 public interface OrderRepository extends JpaRepository<Order, Long> {
     @Query("SELECT EXISTS (" +
-            "SELECT 1 FROM Order o INNER JOIN OrderItem oi ON o.id=oi.orderId " +
+            "SELECT 1 FROM Order o INNER JOIN o.orderItems oi " +
```

Comment:
Is this change intentional?

Reply:
The old statement is wrong HQL syntax when start application has an error occurs so I have changed the to correct syntax 
```
Error interpreting query [SELECT EXISTS (SELECT 1 FROM Order o INNER JOIN OrderItem oi ON o.id=oi.orderId WHERE o.createdBy=:createdBy        AND o.orderStatus=COMPLETED        AND oi.productId in :productId)]; this may indicate a semantic (user query) problem or a bug in the parser [SELECT EXISTS (SELECT 1 FROM Order o INNER JOIN OrderItem oi ON o.id=oi.orderId WHERE o.createdBy=:createdBy        AND o.orderStatus=COMPLETED        AND oi.productId in :productId)]
```